README.md

```diff
@@ -39,52 +39,78 @@ First, you need to have Python 3.6 installed. If you don't, we recommend you to
   sudo add-apt-repository ppa:deadsnakes/ppa
   sudo apt update
   sudo apt install python3.6 python3.6-dev python3.6-pip build-essential
-  pip install -U pipenv
+  pip install -U poetry
+  ```
+
+  optionally install RocksDB lib:
+
+  ```
+  sudo apt install librocksdb-dev
   ```
 - on macOS:
 
   ```
   brew install pyenv
-  pyenv install 3.6.10
-  pyenv local 3.6.10
-  pip install -U pipenv
+  pyenv install 3.6.12
+  pyenv local 3.6.12
+  pip install -U poetry
+  ```
+
+  optionally install RocksDB lib:
+
+  ```
+  sudo apt install librocksdb-dev
   ```
 - on Windows 10 (using [winget](https://github.com/microsoft/winget-cli)):
 
   ```
   winget install python-3.6
-  pip install -U pipenv
+  pip install -U poetry
   ```
 
-### Clone the project and install pipenv dependencies
+  currently it isn't possible to use RocksDB, if you're interested, [please open an issue][open-issue] or if you were
+  able to do this [please create a pull-request with the required steps][create-pr].
```

Comment:
What do you mean? Can't use rocksdb where?

Reply:
I meant to say "it isn't possible to use RocksDB on Windows".

Reply:
Ah, makes sense. The title above it was removed, but I got confused.